src/loader-components/clinical-lowlight-theme/ExampleTemplate.module.scss

```diff
@@ -8,5 +8,8 @@
     --terra-dev-site-example-template-code-background-color: #1d2427;
     --terra-dev-site-example-template-keyboard-focus-outline: #fff;
     --terra-dev-site-example-template-code-toggle-color: #b2b5b6;
+    --terra-dev-site-footer-background-color: #004c76;
+    --terra-dev-site-header-background-color: #004c76;
+    --terra-dev-site-template-content-background-color: #1b2326;
```

Comment:
Were these theme variables named correctly in the original style sheet? I would have expected these be prefixed ` --terra-dev-site-example-template-`?

Reply:
updated here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8